adcgen/indices.py

```diff
@@ -316,6 +320,8 @@ def generic_indices_from_space(space_str: str) -> list[Index]:
     assert len(generic_idx) <= 2  # only occ and virt
     occ = generic_idx.get(("occ", ""), [])
     occ.extend(generic_idx.get(("virt", ""), []))
+    occ.extend(generic_idx.get(("core", ""), []))
+    occ.extend(generic_idx.get(("aux", ""), []))
```

Comment:
Here we can only have occ and virt indices, because we generate indices for a string like 'pphh', where p and h directly map to virtual and occupied indices, respectively.